packages/redux/src/reducers/content.ts

```diff
@@ -36,11 +50,175 @@ export function itemsReducer(state = {
   }
 }
 
-export function navigationReducer(state = {
-  loading: {},
+var currentDescriptorUrl;
+export function descriptorsReducer(state = {
+  loading: {}, // { all: boolean, [id: string]: boolean }
   entries: {}
-}, action: AnyAction) {
+}, action: AnyAction): StateContainer<Item> {
```

Comment:
Item (from @craftercms/models) interface needs to be spelled. Currently it's just the shell.